src/components/Capture/index.js

```diff
@@ -27,6 +30,30 @@ export default class Capture extends Component {
 
   componentDidMount () {
     events.on('onMessage', (message) => this.handleMessages(message))
+    this.checkWebcamSupport();
+  }
+
+  checkWebcamSupport () {
+    DetectRTC.load( _ => {
+      this.setState({
+        DetectRTCLoading: false,
+        hasWebcam: DetectRTC.hasWebcam
+      })
+    });
+  }
+
+  supportsWebcam (){
+    if (DetectRTC.isGetUserMediaSupported && this.state.DetectRTCLoading) return true;
+    return this.state.hasWebcam;
+  }
+
+  //Fired when there is an active webcam feed
+  onUserMedia() {
+    this.setState({
+      hasWebcam: true,
+      hasWebcamPermission: true,
+      DetectRTCLoading: false
+    })
```

Comment:
I have added lint into the build, but there is no semicolon rule on it. I will put a task in the backlog for adding a rule on semicolons.